test/integration/062_defer_state_test/test_modified_state.py

```diff
@@ -202,3 +202,10 @@ def test_postgres_changed_exposure(self):
         results, stdout = self.run_dbt_and_capture(['run', '--models', '+state:modified', '--state', './state'])
         assert len(results) == 1
         assert results[0].node.name == 'view_model'
+
+    @use_profile('postgres')
+    def test_postgres_previous_version_manifest(self):
+        # This tests that a different schema version in the file throws an error
+        with self.assertRaises(IncompatibleSchemaException) as exc:
+            results = self.run_dbt(['ls', '-s',  'state:modified',  '--state',  './previous_state'])
+            self.assertRegex(str(exc), r"Expected a schema version")
```

Comment:
Should this check for the exception code (10014) instead of the message string?

Reply:
Yes, that's better. None of our tests did it that way, of course :)

I changed this test.